api-docs/swagger-v1.1.yml

```diff
@@ -1120,6 +1143,13 @@ parameters:
     minimum: 1
     maximum: 10
     required: false
+  coded:
+    name: coded
+    type: boolean
+    in: query
+    description: restrict to nutrients having a code
+    enum: [true]
```

Comment:
Nitpick: net als bij `scored` misschien onder het type? In zekere zin is dit deel van het type.